src/linear_algebra/dimension.lean

```diff
@@ -38,21 +38,21 @@ variables {ι : Type w} {ι' : Type w'} {η : Type u₁'} {φ : η → Type*}
 
 open_locale classical big_operators
 
-section vector_space
-variables [field K] [add_comm_group V] [vector_space K V] [add_comm_group V₁] [vector_space K V₁]
+section module
+variables [field K] [add_comm_group V] [module K V] [add_comm_group V₁] [module K V₁]
 include K
 open submodule function set
 
 variables (K V)
 
 /-- the dimension of a vector space, defined as a term of type `cardinal` -/
-def vector_space.dim : cardinal :=
+def module.dim : cardinal :=
```

Comment:
This definition can stay in the `vector_space` namespace even though there is no definition called `vector_space` anymore, right?

```suggestion
def vector_space.dim : cardinal :=
```